keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all operations. For more information, see JsonWebKeyOperation.",
```

Comment:
At Pooneh's suggestion, I'm dropping the list of values from the description. This will allow for minor future changes.